styleguide/screens/Patterns/Modifiers/Modifiers.js

```diff
@@ -0,0 +1,425 @@
+          */
+          .fontSmI
+          /*
+            font-family: var(--font-avenir);
+            font-size: var(--fontsize-small-ii);
+            line-height: var(--lineheight-small-ii);
+            letter-spacing: var(--letter-spacing-small-ii);
+            font-weight: var(--fontweight-regular);
+          */
+          .fontSmIi
+        ` }
+        syntax="css"
+      />
+      <T elm="p" className={ m.mtr }>
+        At some points, you may just want to apply <C>font-size</C> to an element.
+        If you don’t want to compose your own class, you can use the following modifiers:
+      </T>
+      <CodeBlock
+        className={ m.mtr }
+        code={ dedent`
+          // --fontsize-large-v
+          .largeV
+          // --fontsize-large-vi
+          .largeIv
+          // --fontsize-large-iii
+          .largeIii
+          // --fontsize-large-ii
+          .largeIi
+          // --fontsize-large-i
+          .largeI
+          // --fontsize-regular
+          .regular
+          // --fontsize-small-i
+          .smallI
+          // --fontsize-small-ii
+          .smallIi
+        ` }
+        syntax="css"
+      />
+      <T elm="p" className={ m.mtr }>
+        Similarly, you can provide the letter spacing that is associated to each font size:
+      </T>
+      <CodeBlock
+        className={ m.mtr }
+        code={ dedent`
+          // --letter-spacing-large-v
+          .letSpacingLargeV
+          // --letter-spacing-large-iv
+          .letSpacingLargeIv
+          // --letter-spacing-large-iii
+          .letSpacingLargeIii
+          // --letter-spacing-large-ii
+          .letSpacingLargeIi
+          // --letter-spacing-large-i
+          .letSpacingLargeI
+          // --letter-spacing-regular
+          .letSpacingRegular
+          // --letter-spacing-small-i
+          .letSpacingSmallI
+          // --letter-spacing-small-ii
+          .letSpacingSmallIi
+        ` }
+        syntax="css"
+      />
+      <T elm="p" className={ m.mtr }>
+        Apply additional typographic styles using various classes which apply weights, styles,
+        decorations and transforms:
+      </T>
+      <CodeBlock
+        className={ m.mtr }
+        code={ dedent`
+          // font-weight: var(--fontweight-bold);
+          .bold
+          // font-weight: var(--fontweight-demi);
+          .demi
+          // font-style: italic;
+          .italic
+          // text-decoration: line-through;
+          .strike
+          // font-family: monospace;
+          .code
+          // text-decoration: underline;
+          .underline
+          // text-transform: uppercase;
+          .uppercase
+          // text-transform: none;
+          .normal
+          // white-space: normal;
+          .wrap
+          // white-space: nowrap;
+          .nowrap
+          // text-align: left;
+          .left
+          // text-align: center;
+          .center
+          // text-align: right;
+          .right
```

Comment:
This gets very hard to read without proper syntax highlighting, maybe it just needs some whitespace